test/vm_tests/builder/parsing/member_association_test.dart

```diff
@@ -22,12 +22,12 @@ import 'parsing_helpers.dart';
 main() {
   group('Member Association', () {
     BoilerplateMemberHelper memberHelper;
-    List<BoilerplateFactory> factories;
-    List<BoilerplateComponent> components;
-    List<BoilerplateProps> props;
-    List<BoilerplateState> states;
-    List<BoilerplateStateMixin> stateMixins;
-    List<BoilerplatePropsMixin> propsMixins;
+    List<BoilerplateFactory>? factories;
+    List<BoilerplateComponent>? components;
+    List<BoilerplateProps>? props;
+    List<BoilerplateState>? states;
+    List<BoilerplateStateMixin>? stateMixins;
+    List<BoilerplatePropsMixin>? propsMixins;
```

Comment:
If these were moved within the group that starts on line 95 - could they be made `late` non-nullable?

Reply:
Good catch; was able to clean up this and related tests quite a bit: https://github.com/Workiva/over_react/pull/856/files/8ded99daada4aa5a0c6f8e3066a0ffed56ea7e5c..92f7e5adf70a987c2882d74004f620ab86c2741e